apps/omg/lib/omg/output.ex

```diff
@@ -41,25 +41,29 @@ defmodule OMG.Output do
   def reconstruct([raw_type, rlp_decoded_chunks]) when is_binary(raw_type) do
     case RawData.parse_uint256(raw_type) do
       {:ok, output_type} when output_type in @output_types ->
-        reconstruct([output_type, rlp_decoded_chunks])
+        do_reconstruct_fungible([output_type, rlp_decoded_chunks])
 
       {:ok, _unrecognized_type} ->
         {:error, :unrecognized_output_type}
     end
   end
 
-  @doc """
-  Reconstructs the structure from a list of RLP items
-  """
-  def reconstruct([output_type, [owner_rlp, currency_rlp, amount_rlp]]) do
+  def reconstruct([_raw_type, _rlp_decoded_chunks]), do: {:error, :unrecognized_output_type}
+
+  # Reconstructs the structure from a list of RLP items
+  #
+  # TODO: after the `OMG.Output` refactors, `reconstruct/1` conflated decoding of valid
```

Comment:
I'm starting to think TODO's are a bad practice (wink @unnawut)
can you elaborate what this TODO means? if it's required it might be better to create an issue, if it's not required... delete it?

Reply:
hm, I decided for a `TODO` specifically here, since this intersects with the in-progress `ex_plasma` refactor. The intention is that whoever pushes those refactors forward bumps into this when working on this code. Agreed that an issue is generally better, but wouldn't provide the "bump into". This is `TODO` should be short-lived.

Reply:
actually, it turns out, this already is gone due to one of the PRs going in first, resolving